tox.ini

```diff
@@ -1,32 +1,32 @@
 [tox]
 envlist =
-    py27,py33,py34,py35,py36,pypy,pypy3,coverage,docs
+    py27,py34,py35,py36,pypy,pypy3
```

Comment:
- I'll restore docs.
- With a tiny tweak, we can get 100% coverage in any given environment, making the complicated dance that the `coverage` environment is doing and the need for it to run after everything else has obsolete. That would also eliminate the need for it to not be listed. Let me try that.